Clean up debug webhooks guide (#977)
clerk/clerk-docs: docs/integrations/webhooks/debug-your-webhooks.mdx

If you are having trouble with your webhook, you can create a basic Route Handler and then test it locally.  First, add the following file and code to your app.

> If you are having trouble with your webhook, you can create a basic response and then test it locally.

Reviewer: ```suggestion
If you'e having trouble with your webhook, you can create a basic response and then test it locally.
```

The results of these will appear in the command line where you are running your application.

> Checking the values and the progress of the webhook code will allow you to narrow down where the code is failing. The will often return `null` or errors where they should be returning values.

Reviewer: ```suggestion
Checking the values and the progress of the webhook code will allow you to narrow down where the code is failing. They will often return `null` or errors where they should be returning values.
```

## Check your logic

Once you have verified the webhook, you will now be writing your own code to handle the values from the webhook. This could range from saving data to a databse, or integrating with another system, to updating users or sending emails or SMS.

> If the webhook is verified and you are seeing a `500` status code or your webhook is not behaving as expected, remember that you can use `console.log` to help diagnose what the problem is. Console logs and error will be displayed on your command line, allowing you to see what's happening and address the bugs.

Reviewer: ```suggestion
If the webhook is verified and you're seeing a `500` status code or your webhook is not behaving as expected, remember that you can use `console.log` to help diagnose what the problem is. Console logs and errors will be displayed on your command line, allowing you to see what's happening and address the bugs.
```
